bin/nomplate.js

```diff
@@ -33,11 +39,11 @@ function processArgKey(key) {
   while (key[0] === '-') {
     key = key.slice(1);
   }
-
   return key;
 }
 
 function processHandlerArgs(args) {
+
```

Comment:
I generally appreciate these insertions too, but the linter will complain (once it's hooked up properly with CI) that there shouldn't be unused whitespace at the beginning or end of a block.